src/RelationParser.h

```diff
@@ -0,0 +1,74 @@
+//
+//  RelationParser.h
+//  snowcrash
+//
+//  Created by Pavan Kumar Sunkara on 04/03/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#ifndef SNOWCRASH_RELATIONPARSER_H
+#define SNOWCRASH_RELATIONPARSER_H
+
+#include "SectionParser.h"
+#include "RegexMatch.h"
+
+namespace snowcrash {
+
+    /** Link Relation matching regex */
+    const char* const RelationRegex = "^[[:blank:]]*[Rr]elation[[:blank:]]*:" SYMBOL_IDENTIFIER "?$";
```

Comment:
@pksunkara 

So according to [RFC 5988](https://tools.ietf.org/html/rfc5988) a relation type should be

```
  relation-types = relation-type
                 | <"> relation-type *( 1*SP relation-type ) <">
  relation-type  = reg-rel-type | ext-rel-type
  reg-rel-type   = LOALPHA *( LOALPHA | DIGIT | "." | "-" )
  ext-rel-type   = URI
```

So Snow Crash should really conform to this. Also in respect to what @klokane mentions it should be implemented with https://github.com/apiaryio/snowcrash/milestones/Keywords%20&%20Identifiers in mind. 

That is:

1. parser should warn when it encounters `+ Relation` anywhere except an action section
2. parser should recognize any `+ [Rr]elation\s+([.])* and then work with the capture group

Does this answer your question?